CourseGrab/Home/HomeTableViewHeader.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  HomeTableViewHeader.swift
+//  CourseGrab
+//
+//  Created by Mathew Scullin on 2/4/20.
+//  Copyright © 2020 Cornell AppDev. All rights reserved.
+//
+
+import UIKit
+import SnapKit
+
+class HomeTableViewHeader: UITableViewHeaderFooterView {
+
+    private let title = UILabel()
+
+    override init(reuseIdentifier: String?) {
+        super.init(reuseIdentifier: reuseIdentifier)
+
+        contentView.backgroundColor = .white
+        title.font = ._20Semibold
+        contentView.addSubview(title)
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    private func setupContraints (section: Int) {
+        let padding = section == 0 ? 24 : 6
+        title.snp.makeConstraints { make in
```

Comment:
because we're updating the constraints inside the `configure` (i.e. we're reusing this header view), we should be using `remakeConstraints` instead of `makeConstraints` because it can be called multiple times